packages/react/src/select/root/SelectRoot.tsx

```diff
@@ -32,6 +32,7 @@ const SelectRoot: SelectRoot = function SelectRoot<Value>(
     disabled = false,
     readOnly = false,
     required = false,
+    modal = false,
   } = props;
```

Comment:
The problem with select modality is that it prevents touch screen readers from being able to escape it without selecting anything.